docs/setup_vscode.md

```diff
@@ -71,70 +71,55 @@ $ code --version
 x64
 ```
 
-#### WSL remote mode
-
-Use VS Code's [remote mode](https://code.visualstudio.com/docs/remote/wsl) to connect the VS Code graphical user interface (GUI) running on Windows to the Linux environment and tools like `g++` running on WSL.
-
-WSL Remote Mode connects VS Code to a remote instance of a VS Code server running elsewhere. When you launch VS Code from the WSL terminal, a VS Code server is started within WSL and the VS Code UI running on Windows connects to that server.
-
-<img src="images/vscode065.png" width="512px">
-
-Install the [WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl).  It's OK if you have other extensions installed.
-```console
-$ code --install-extension ms-vscode-remote.remote-wsl
-$ code --list-extensions
-ms-vscode-remote.remote-wsl
-```
-
-After the extension is installed, quit VS Code and start it again.
-
-You'll know that VS Code is running in remote mode when you see the remote mode indicator in the bottom left corner.
-
-<img src="images/vscode068.png" width="768px">
-
-<div class="primer-spec-callout warning" markdown="1">
-**Pitfall:** If you accidentally open VS Code from Windows mode, click on the green icon in the lower left hand corner and then select "Reopen Folder in WSL".
```

Comment:
I think this got cut unintentionally.

Reply:
Actually, I just realized the pictures were reworked. That's fine, but we could add back in some text that specifies the fix to click it and select "Reopen Folder in WSL"? Otherwise students won't know how to resolve it.

Reply:
I agree with adding back the text here.

Reply:
I removed this because I copied it to [here](https://eecs280staff.github.io/tutorials/setup_vscode.html#create-launchjson) yesterday.  Then, today, I realized that it doesn't make sense in the install step because students don't have any files yet.

Reply:
Ah, got it!